src/js/controllers/index.js

```diff
@@ -274,6 +278,7 @@ angular.module('copayApp.controllers').controller('indexController', function($r
         self.walletScanStatus = walletStatus.wallet.scanStatus;
         self.copayers = walletStatus.wallet.copayers;
         self.preferences = walletStatus.preferences;
+        $rootScope.$emit('Local/BalanceUpdated', walletStatus.balance);
```

Comment:
OK. But to do a code more clean I ‘d like to add a comment before like this:

```
// Used for external plugin or something
$rootScope.$emit('Local/BalanceUpdated', walletStatus.balance);

// Status Shortcuts
... 
```

That's all. Everything LGTM.